Implement tree navigation (CHARTS-197)

ripla opened the pull request:
<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/vaadin/charts-component/32)
<!-- Reviewable:end -->

alvarezguille commented:
Reviewed 6 of 8 files at r1.
Review status: 6 of 8 files reviewed at latest revision, 3 unresolved discussions, all commit checks successful.

---

<sup>**[demo/menu-tree.html, line 99 \[r1\]](https://reviewable.io:443/reviews/vaadin/charts-component/32#-Jx_P5eWk_afFcnN8sF2)** ([raw file](https://github.com/vaadin/charts-component/blob/9968abe43192944aad8980c7b6b26ad3914971bb/demo/menu-tree.html#L99)):</sup>
needed?

---

<sup>**[demo/menu-tree.html, line 111 \[r1\]](https://reviewable.io:443/reviews/vaadin/charts-component/32#-Jx_OoHkwZ3yIydCWW1B)** ([raw file](https://github.com/vaadin/charts-component/blob/9968abe43192944aad8980c7b6b26ad3914971bb/demo/menu-tree.html#L111)):</sup>
unused?

---

<sup>**[v-series.html, line 84 \[r1\]](https://reviewable.io:443/reviews/vaadin/charts-component/32#-Jx_JHTQSy-BnmQEqU7j)** ([raw file](https://github.com/vaadin/charts-component/blob/9968abe43192944aad8980c7b6b26ad3914971bb/v-series.html#L84)):</sup>
remove commented code

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/charts-component/32)
<!-- Sent from Reviewable.io -->

ripla commented:
Review status: 6 of 8 files reviewed at latest revision, 3 unresolved discussions, all commit checks successful.

---

<sup>**[demo/menu-tree.html, line 99 \[r1\]](https://reviewable.io:443/reviews/vaadin/charts-component/32#-Jx_P5eWk_afFcnN8sF2)** ([raw file](https://github.com/vaadin/charts-component/blob/9968abe43192944aad8980c7b6b26ad3914971bb/demo/menu-tree.html#L99)):</sup>
No, will remove.

---

<sup>**[demo/menu-tree.html, line 111 \[r1\]](https://reviewable.io:443/reviews/vaadin/charts-component/32#-Jx_OoHkwZ3yIydCWW1B)** ([raw file](https://github.com/vaadin/charts-component/blob/9968abe43192944aad8980c7b6b26ad3914971bb/demo/menu-tree.html#L111)):</sup>
Yes, will remove.

---

<sup>**[v-series.html, line 84 \[r1\]](https://reviewable.io:443/reviews/vaadin/charts-component/32#-Jx_JHTQSy-BnmQEqU7j)** ([raw file](https://github.com/vaadin/charts-component/blob/9968abe43192944aad8980c7b6b26ad3914971bb/v-series.html#L84)):</sup>
Removed, updated code comment.

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/charts-component/32)
<!-- Sent from Reviewable.io -->

alvarezguille commented:
Reviewed 1 of 8 files at r1, 3 of 3 files at r2.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/charts-component/32)
<!-- Sent from Reviewable.io -->
